go.mod

```diff
@@ -63,7 +63,7 @@ require (
 	github.com/iamacarpet/go-win64api v0.0.0-20210311141720-fe38760bed28
 	github.com/k3s-io/helm-controller v0.10.5
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/k3s v1.21.1-rc1.0.20210817230230-e32292478107
+	github.com/rancher/k3s v1.21.1-rc1.0.20210820193422-ed5991f13b14 // engine-1.21
```

Comment:
Hmm, have/should we consider(ed) tagging this so that `engine-1.21` is in the tag somewhere to foil `go mod` from misleadingly picking the nearest proximate tag?

Reply:
done!